.eslintrc.js

```diff
@@ -1,57 +1,19 @@
 'use strict';
 
+const { configs } = require('@nullvoxpopuli/eslint-configs');
```

Comment:
perhaps it would simpler to keep the config boilerplate here and avoid the indirection?

Reply:
I don't simpler, but def clearer as to what's happening. Linting in hybrid projects unfortunately isn't all that simple anymore :(

If you're curious though, here is the config:
https://github.com/NullVoxPopuli/eslint-configs/blob/main/configs/ember.js#L141

Reply:
👍 thanks. Agree it's not simpler, but perhaps clearer.

It'll only become an issue if we need to modify this at some point, that may never happen though. Happy to ship this as is and revisit if that ever happens

Reply:
So _far_, modifying has been easy, as demonstrated here: https://github.com/NullVoxPopuli/limber/blob/main/frontend/.eslintrc.js#L14 (which, this specific modification is now obsolete). 

But yeah, I hear you. In this eslint-configs package, I should probably add a `bin` to "eject" the config, kinda like create-react-app does for all the webpack configs 🤔